rwgame/states/DebugState.cpp

```diff
@@ -33,7 +33,7 @@ static void jumpCharacter(RWGame* game, CharacterObject* player,
     }
 }
 
-std::shared_ptr<Menu> DebugState::createDebugMenu() {
+std::optional<Menu> DebugState::createDebugMenu() {
```

Comment:
Can't these function (and anywhere else) return a plain `Menu` (without the `std::optional`-wrapping)?

Reply:
Can, was used that version earlier. I've changed because my ide says it can prevent copy elision on some compilers. (qt creator) Revert?

Reply:
Thanks for the clarification. You don't have to revert. Ideally, the compiler should be smart enough. Would lto optimize this?

Reply:
I don't have enough knowledge about lto to answer. :(